### DISCUSSION TITLE
Welcome to Giphy_Explorer Discussions!

### DISCUSSION THREAD

**Discussion by lcrojano** [lcrojano/Giphy_Explorer#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lonewolf9277** [comment]
Hello I noticed that the api key is exposed in the script file. Shouldn’t this api part be moved to the server? There could be a simple flask backend.

**Reply by lcrojano** [reply]
Thanks for pointing it out. Would you like to work on this issue? 
https://github.com/lcrojano/Giphy_Explorer/issues/57

**Reply by lonewolf9277** [reply]
Sure
